netmanager/src/config/constants.js

```diff
@@ -195,8 +193,6 @@ const devConfig = {
   DEVICE_MAINTENANCE_LOG_URI:
     "http://localhost:4001/api/v1/device/monitor/maintenance_logs/",
   //DELETE_DEVICE_URI: "http://127.0.0.1:3000/api/api/v1/devices/ts/delete?device="
-  GET_ONLINE_OFFLINE_MAINTENANCE_STATUS:
```

Comment:
Same here, we need to bring these constants back and just change environments using appropriate runtime commands (`npm run pc-dev`, etc) depending on your machine and the environment you would like to carry out tests in.

Reply:
this is already in staging.

Reply:
True, you are right. This is already in staging but this PR wants to remove them.